### DISCUSSION TITLE
metadata dialog not showing automatically when exporting to flac - also "Show Metadata Editor prior to export step" preference missing

### DISCUSSION THREAD

**Discussion by toby11** [audacity/audacity#7361] [Development]
### Bug description

I have just updated to 3.6.3 and on saving flac files the metadata dialog no longer shows automatically.

### Steps to reproduce

Using Files - Export Audio the metadata window does not show automatically anymore - from looking this up it appears you used to be able to use this prference "Show Metadata Editor prior to export step" - but that seems to no long exist?

### Expected behavior

Show metadata dialog before export step.

### Actual behavior

The dialog does not show.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by petersampsonaudacity** [comment]
Muse changed that - the Metadata editor is now accessed in the Export dialog by pressing the **Edit Metadata** button.

Please be aware there is a metadata bug (logged, but as yet remains un-fixed) when using Export Multiple:

 **With new Multiple Files Export dialog user supplied Metadata applies only to the first exported file - and does not propagate to iTunes on import** #5825 

Peter

**Comment by toby11** [comment]
thanks - so you cannot make it show automatically on every save now? Seem odd to remove it (even with a preference setting)

I'll revert back to the older version I was using I think.
